### DISCUSSION TITLE
Account verification

### DISCUSSION THREAD

**Discussion by aropan** [aropan/clist#247] [News]
https://github.com/aropan/clist/assets/1968460/d55480aa-fd4f-45d5-80f8-cb94176102d2

[Try](https://clist.by/coder/) for your accounts.

**Comment by michelececcacci** [comment]
Oh, thanks a lot. I really appreciate it. I couldn't get it to work on LC, but it was amazing for CF

**Reply by aropan** [reply]
@michelececcacci I rechecked it and it works. I see ten other coders who verified their leetcode accounts. So If you don't understand, then let me know.

**Reply by michelececcacci** [reply]
Ok, I'll try again later. Thanks a lot.
